cmd/minikube/cmd/start_flags.go

```diff
@@ -636,7 +643,7 @@ func generateNewConfigFromFlags(cmd *cobra.Command, k8sVersion string, rtime str
 			APIServerName:          viper.GetString(apiServerName),
 			APIServerNames:         apiServerNames,
 			APIServerIPs:           apiServerIPs,
-			DNSDomain:              viper.GetString(dnsDomain),
```

Comment:
what happens to "dnsDomain" variable, is it used anywhere else ? should that be updated too